examples/timestamp/signed_timestamp.c

```diff
@@ -337,27 +219,20 @@ int TPM2_Timestamp_Test(void* userCtx)
     }
 
     /* Close key handles */
-    if (rsaKey.handle != TPM_RH_NULL) {
-        cmdIn.flushCtx.flushHandle = rsaKey.handle;
+    if (rsaKey.handle.hndl != TPM_RH_NULL) {
```

Comment:
These can be refactored to use `wolfTPM2_UnloadHandle(&dev, &rsaKey.handle);`

Reply:
Good idea ! I like it